src/core/unit_tests/CMakeLists.txt

```diff
@@ -68,3 +68,4 @@ unit_test(NAME field_coupling_force_field SRC field_coupling_force_field_test.cp
 unit_test(NAME Span_test SRC Span_test.cpp)
 unit_test(NAME periodic_fold_test SRC periodic_fold_test.cpp)
 unit_test(NAME sendrecv_test SRC sendrecv_test.cpp DEPENDS Boost::mpi MPI::MPI_CXX)
+unit_test(NAME PdbParser_test SRC PdbParser_test.cpp ../PdbParser.cpp)
```

Comment:
Actually I meant to create a new target for the PDB parser since it does not depend on anything else in the core. I think it's not the cleanest solution to use the relative path to the source file here but depend on the to be created PDB target.